lib/game/player/easy_computer_player.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Game.Player.EasyComputerPlayer do
+  @moduledoc """
+  The module that handles an Easy Computer Players move
+  """
+
+  alias Game.Player.PlayerBehaviour
+  @behaviour PlayerBehaviour
+
+  @impl PlayerBehaviour
+  def valid_input(board, communicator) do
+    valid_move = board
+    |> Map.filter(fn {_key, val} -> val == :empty end)
+    |> Map.keys()
+    |> List.first()
+    communicator.display("please make desired move (Computer): #{valid_move}\n")
```

Comment:
Do we need to ask the computer for a move?

Reply:
I had it display this for consistencies sake when it came to the differences between displaying a human move and a computer move. Like I included it more for readability of the user than anything, I can remove it if need be or that is preferred!